### ISSUE TITLE
Information Dialog

### ISSUE THREAD

**Issue by kimohoki** [ModOrganizer2/modorganizer#874]
## The problem:
Everytime I try to open an information dialog box for any given mod in Mod Organizer v2.2.1 there is no dialog box and Mod Organizer becomes unresponsive. The program must then be forced closed through task manager. This is a new issue as I have not encountered this before and I have been using Mod Organizer for a few years now.

## Environment:

* Mod Organizer Version that exhibits the issue:
* Last Mod Organizer Verison that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Mod Organizer:

## Details:
If necessary, describe the problem you have been experiencing with more details and pictures.

## Link to Mod Organizer logs:
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder". You should close MO2 before sending or examining the logs as they are finished writing only after the program is closed.

### USVFS:
Create a [GIST](https://gist.github.com) which is a paste of your USVFS logs, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the USVFS logs!

### MO Interface:
Create a [GIST](https://gist.github.com) which is a paste of your mo_interface logs, and link them here.
Do _NOT_ paste your full mo_interface logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the mo_interface logs!

**Comment by LostDragonist** [comment]
Make the issue happen then press Alt-Space, M, and left or right arrow.  Or maybe whatever the maximize key is.

**Comment by kimohoki** [comment]
Nothing like that seems to be working for me.

**Comment by LostDragonist** [comment]
In that case, please post your mo_interface.log per the issue template.

**Comment by kimohoki** [comment]
18:33:49 [D] Starting Mod Organizer version "2.2.1" revision e69078e
18:33:49 [D] preloading "libcrypto-1_1-x64.dll"
18:33:49 [D] preloading "libssl-1_1-x64.dll"
18:33:49 [D] ssl support: 1
18:33:50 [D] windows: version: 10.0.18362, 18362.418, release 1903, 18362.1.amd64fre.19h1_release.190318-1202, Windows 10 Home, elevated: yes
18:33:50 [D] security products:
18:33:50 [D]  . Spybot - Search and Destroy (autoupdate|antivirus), active, definitions outdated
18:33:50 [D]  . Avast Antivirus (antivirus), inactive
18:33:50 [D]  . Avast Antivirus (antivirus), active
18:33:50 [D]  . Avast Antivirus (antivirus), active
18:33:50 [D]  . Windows Defender (autoupdate|antivirus), inactive
18:33:50 [D]  . Windows Firewall (firewall), active
18:33:50 [D] modules loaded in process:
18:33:50 [D]  . c:/windows/system32/advapi32.dll, 667648 B, 6.2.18362.329, 10.0.18362.1 (WinBuild.160101.0800), 2019-09-11T16:23:27
18:33:50 [D]  . c:/windows/system32/apphelp.dll, 585728 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:28
18:33:50 [D]  . c:/windows/system32/bcrypt.dll, 155648 B, 6.2.18362.267, 10.0.18362.1 (WinBuild.160101.0800), 2019-08-22T23:34:15
18:33:50 [D]  . c:/windows/system32/bcryptprimitives.dll, 524288 B, 6.2.18362.295, 10.0.18362.295 (WinBuild.160101.0800), 2019-08-22T23:34:18
18:33:50 [D]  . c:/windows/system32/cfgmgr32.dll, 303104 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:39:44
18:33:50 [D]  . c:/windows/system32/combase.dll, 3366912 B, 6.2.18362.356, 10.0.18362.1 (WinBuild.160101.0800), 2019-09-11T16:24:03
18:33:50 [D]  . c:/windows/system32/comdlg32.dll, 851968 B, 6.2.18362.418, 10.0.18362.1 (WinBuild.160101.0800), 2019-10-09T22:42:32
18:33:50 [D]  . c:/windows/system32/crypt32.dll, 1347584 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:33
18:33:50 [D]  . c:/windows/system32/cryptbase.dll, 49152 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:36
18:33:50 [D]  . c:/windows/system32/cryptsp.dll, 94208 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:33
18:33:50 [D]  . c:/windows/system32/d3d11.dll, 2469888 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:39:33
18:33:50 [D]  . c:/windows/system32/d3d9.dll, 1863680 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:39:53
18:33:50 [D]  . c:/windows/system32/dbgcore.dll, 172032 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/dbghelp.dll, 2048000 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/devobj.dll, 172032 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:39:44
18:33:50 [D]  . c:/windows/system32/dhcpcsvc.dll, 114688 B, 6.2.18362.267, 10.0.18362.1 (WinBuild.160101.0800), 2019-08-22T23:34:15
18:33:50 [D]  . c:/windows/system32/dnsapi.dll, 827392 B, 6.2.18362.267, 10.0.18362.1 (WinBuild.160101.0800), 2019-08-22T23:34:15
18:33:50 [D]  . c:/windows/system32/driverstore/filerepository/nv_dispi.inf_amd64_827405c7c65146ab/nvoglv64.dll, 40079360 B, 26.21.14.3648, 2019-10-04T20:55:38
18:33:50 [D]  . c:/windows/system32/dwmapi.dll, 184320 B, 6.2.18362.267, 10.0.18362.1 (WinBuild.160101.0800), 2019-08-22T23:34:13
18:33:50 [D]  . c:/windows/system32/dwrite.dll, 3137536 B, 6.2.18362.356, 10.0.18362.1 (WinBuild.160101.0800), 2019-09-11T16:23:45
18:33:50 [D]  . c:/windows/system32/dxcore.dll, 131072 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:06
18:33:50 [D]  . c:/windows/system32/dxgi.dll, 962560 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:39:33
18:33:50 [D]  . c:/windows/system32/dxva2.dll, 143360 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:11
18:33:50 [D]  . c:/windows/system32/gdi32.dll, 155648 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:06
18:33:50 [D]  . c:/windows/system32/gdi32full.dll, 1654784 B, 6.2.18362.356, 10.0.18362.356 (WinBuild.160101.0800), 2019-09-11T16:24:12
18:33:50 [D]  . c:/windows/system32/glu32.dll, 180224 B, 6.2.18362.387, 10.0.18362.1 (WinBuild.160101.0800), 2019-10-01T20:40:13
18:33:50 [D]  . c:/windows/system32/hid.dll, 57344 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:50
18:33:50 [D]  . c:/windows/system32/iertutil.dll, 2777088 B, 11.0.18362.387, 11.00.18362.1 (WinBuild.160101.0800), 2019-10-01T20:39:53
18:33:50 [D]  . c:/windows/system32/imagehlp.dll, 118784 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/imm32.dll, 188416 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:39:45
18:33:50 [D]  . c:/windows/system32/iphlpapi.dll, 237568 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:33
18:33:50 [D]  . c:/windows/system32/kernel.appcore.dll, 69632 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:15
18:33:50 [D]  . c:/windows/system32/kernel32.dll, 729088 B, 6.2.18362.329, 10.0.18362.329 (WinBuild.160101.0800), 2019-09-11T16:23:29
18:33:50 [D]  . c:/windows/system32/kernelbase.dll, 2764800 B, 6.2.18362.418, 10.0.18362.329 (WinBuild.160101.0800), 2019-10-09T22:42:23
18:33:50 [D]  . c:/windows/system32/mpr.dll, 110592 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:36
18:33:50 [D]  . c:/windows/system32/msasn1.dll, 73728 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/msctf.dll, 1269760 B, 6.2.18362.387, 10.0.18362.1 (WinBuild.160101.0800), 2019-10-01T20:39:43
18:33:50 [D]  . c:/windows/system32/msdmo.dll, 45056 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:43:47
18:33:50 [D]  . c:/windows/system32/msvcp140.dll, 684032 B, 14.13.26118.0, 14.13.26118.0 built by: VCTOOLSREL, 2019-05-01T22:08:48
18:33:50 [D]  . c:/windows/system32/msvcp_win.dll, 647168 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:39:43
18:33:50 [D]  . c:/windows/system32/msvcrt.dll, 647168 B, 7.0.18362.1, 7.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/ncrypt.dll, 155648 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/netapi32.dll, 94208 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:00
18:33:50 [D]  . c:/windows/system32/netutils.dll, 49152 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/nsi.dll, 32768 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:36
18:33:50 [D]  . c:/windows/system32/ntasn1.dll, 241664 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/ntdll.dll, 2031616 B, 6.2.18362.418, 10.0.18362.329 (WinBuild.160101.0800), 2019-10-09T22:42:22
18:33:50 [D]  . c:/windows/system32/ntmarta.dll, 200704 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/nvspcap64.dll, 2887680 B, 3.20.0.118, 2019-09-07T22:31:50
18:33:50 [D]  . c:/windows/system32/ole32.dll, 1400832 B, 6.2.18362.113, 10.0.18362.1 (WinBuild.160101.0800), 2019-08-22T23:34:13
18:33:50 [D]  . c:/windows/system32/oleaut32.dll, 802816 B, 6.2.18362.329, 10.0.18362.329 (WinBuild.160101.0800), 2019-09-11T16:24:03
18:33:50 [D]  . c:/windows/system32/opengl32.dll, 1400832 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:40:13
18:33:50 [D]  . c:/windows/system32/powrprof.dll, 303104 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/profapi.dll, 126976 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:36
18:33:50 [D]  . c:/windows/system32/propsys.dll, 978944 B, 7.0.18362.267, 7.0.18362.1 (WinBuild.160101.0800), 2019-08-22T23:34:05
18:33:50 [D]  . c:/windows/system32/psapi.dll, 32768 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:28
18:33:50 [D]  . c:/windows/system32/rpcrt4.dll, 1179648 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:36
18:33:50 [D]  . c:/windows/system32/rsaenh.dll, 208896 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/sechost.dll, 618496 B, 6.2.18362.267, 10.0.18362.1 (WinBuild.160101.0800), 2019-08-22T23:34:18
18:33:50 [D]  . c:/windows/system32/secur32.dll, 49152 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:45:00
18:33:50 [D]  . c:/windows/system32/setupapi.dll, 4653056 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:45:00
18:33:50 [D]  . c:/windows/system32/shcore.dll, 692224 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:21
18:33:50 [D]  . c:/windows/system32/shell32.dll, 7229440 B, 6.2.18362.387, 10.0.18362.1 (WinBuild.160101.0800), 2019-10-01T20:39:41
18:33:50 [D]  . c:/windows/system32/shlwapi.dll, 335872 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:45:05
18:33:50 [D]  . c:/windows/system32/srvcli.dll, 155648 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:35
18:33:50 [D]  . c:/windows/system32/sspicli.dll, 192512 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:36
18:33:50 [D]  . c:/windows/system32/ucrtbase.dll, 1024000 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:39:43
18:33:50 [D]  . c:/windows/system32/umpdc.dll, 65536 B, (no version), 2019-03-19T04:43:49
18:33:50 [D]  . c:/windows/system32/urlmon.dll, 1925120 B, 11.0.18362.387, 11.00.18362.1 (WinBuild.160101.0800), 2019-10-01T20:39:53
18:33:50 [D]  . c:/windows/system32/user32.dll, 1654784 B, 6.2.18362.418, 10.0.18362.1 (WinBuild.160101.0800), 2019-10-09T22:42:20
18:33:50 [D]  . c:/windows/system32/userenv.dll, 151552 B, 6.2.18362.387, 10.0.18362.1 (WinBuild.160101.0800), 2019-10-01T20:39:44
18:33:50 [D]  . c:/windows/system32/usp10.dll, 102400 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:38
18:33:50 [D]  . c:/windows/system32/uxtheme.dll, 626688 B, 6.2.18362.387, 10.0.18362.1 (WinBuild.160101.0800), 2019-10-01T20:39:43
18:33:50 [D]  . c:/windows/system32/vcruntime140.dll, 90112 B, 14.13.26118.0, 14.13.26118.0 built by: VCTOOLSREL, 2019-05-01T22:08:48
18:33:50 [D]  . c:/windows/system32/version.dll, 40960 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:45:05
18:33:50 [D]  . c:/windows/system32/win32u.dll, 135168 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:39:34
18:33:50 [D]  . c:/windows/system32/windows.storage.dll, 7860224 B, 6.2.18362.387, 10.0.18362.1 (WinBuild.160101.0800), 2019-10-01T20:39:34
18:33:50 [D]  . c:/windows/system32/winhttp.dll, 983040 B, 6.2.18362.387, 10.0.18362.1 (WinBuild.160101.0800), 2019-10-01T20:39:44
18:33:50 [D]  . c:/windows/system32/winmm.dll, 147456 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:43:47
18:33:50 [D]  . c:/windows/system32/winmmbase.dll, 184320 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:43:47
18:33:50 [D]  . c:/windows/system32/winspool.drv, 561152 B, 6.2.18362.267, 10.0.18362.1 (WinBuild.160101.0800), 2019-08-22T23:33:48
18:33:50 [D]  . c:/windows/system32/winsta.dll, 376832 B, 6.2.18362.53, 10.0.18362.53 (WinBuild.160101.0800), 2019-08-22T23:34:15
18:33:50 [D]  . c:/windows/system32/wintrust.dll, 376832 B, 6.2.18362.387, 10.0.18362.387 (WinBuild.160101.0800), 2019-10-01T20:39:34
18:33:50 [D]  . c:/windows/system32/ws2_32.dll, 454656 B, 6.2.18362.387, 10.0.18362.1 (WinBuild.160101.0800), 2019-10-01T20:39:45
18:33:50 [D]  . c:/windows/system32/wtsapi32.dll, 77824 B, 6.2.18362.1, 10.0.18362.1 (WinBuild.160101.0800), 2019-03-19T04:44:33
18:33:50 [D]  . c:/windows/winsxs/amd64_microsoft.windows.common-controls_6595b64144ccf1df_5.82.18362.418_none_2a23d356466d526c/comctl32.dll, 692224 B, 5.82.18362.418, 5.82 (WinBuild.160101.0800), 2019-10-09T22:39:09
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/liblz4.dll, 118784 B, 1.9.1.0, 1.9.1, 2019-09-25T21:48:02, 9210c62b8c005493b63da03e3f9dcd51
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5core.dll, 6197248 B, 5.12.4.0, 2019-09-25T21:48:03, 5214676d935539637ffacbd9ae57fbc3
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5gui.dll, 6549504 B, 5.12.4.0, 2019-09-25T21:48:03, ddcbc3b38c2690e98a6b6e02de4e6bb6
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5network.dll, 1347584 B, 5.12.4.0, 2019-09-25T21:48:04, 1d285046681d203d29d6a0478e35332e
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5positioning.dll, 335872 B, 5.12.4.0, 2019-09-25T21:48:04, 646e55e38e6a036a2d90184430a0c865
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5printsupport.dll, 335872 B, 5.12.4.0, 2019-09-25T21:48:04, b951f20cd610d10d8b82a757cf847453
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5qml.dll, 4112384 B, 5.12.4.0, 2019-09-25T21:48:04, 55865081b20567aa4213832512c91e8f
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5quick.dll, 3825664 B, 5.12.4.0, 2019-09-25T21:48:04, 8a5b9ad37925c51f411b0ae84908e5ca
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5quickwidgets.dll, 90112 B, 5.12.4.0, 2019-09-25T21:48:04, 4648f0e9ae96a170a8cba82dcfa94ea5
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5webchannel.dll, 139264 B, 5.12.4.0, 2019-09-25T21:48:04, 91a2f2301995a89811397b022899b642
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5webenginecore.dll, 81231872 B, 5.12.4.0, 2019-09-25T21:48:04, 447fbe2838359c8509ff3a8583bdbb8c
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5webenginewidgets.dll, 245760 B, 5.12.4.0, 2019-09-25T21:48:07, ce61d88d09e1984ffbf04e25db883472
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5websockets.dll, 167936 B, 5.12.4.0, 2019-09-25T21:48:07, 4941f5a135ae294cd6d3a77c3b816cb3
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5widgets.dll, 5607424 B, 5.12.4.0, 2019-09-25T21:48:07, 5f06de94123c883e2a0db53b3f89b17f
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/dlls/qt5winextras.dll, 585728 B, 5.12.4.0, 2019-09-25T21:48:07, 23d6db8e51f72b3f744cd2afd8e60930
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/libcrypto-1_1-x64.dll, 3436544 B, 1.1.1.3, 1.1.1c, 2019-09-25T21:48:01, 29e94b53eeb818949743033d8914c4dd
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/libssl-1_1-x64.dll, 704512 B, 1.1.1.3, 1.1.1c, 2019-09-25T21:48:01, 7cad81db65047801c62c438e04195858
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/modorganizer.exe, 5304320 B, 2.2.1.0, 2.2.1, 2019-09-25T21:48:01, 672299920c58499866f3e971565502d2
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/platforms/qwindows.dll, 1503232 B, 5.12.4.0, 2019-09-25T21:48:02, 834d11c46c6473af57eb4a202e21a07a
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/styles/qwindowsvistastyle.dll, 155648 B, 5.12.4.0, 2019-09-25T21:48:03, 3c0728fe52fc0822c505cc7f74912ac6
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/uibase.dll, 499712 B, 2.2.0.1, 2019-09-25T21:48:02, ddb1edb9dc2b1d079ca3f71c67713661
18:33:50 [D]  . d:/games/modorganizer2 skyrimse/usvfs_x64.dll, 1855488 B, 0.4.4.0, 2019-09-25T21:48:02, 95d316e1be77ee6929685b5fb3d02ee7
18:33:50 [D] data path: C:/Users/jkleb/AppData/Local/ModOrganizer/SkyrimSE
18:33:50 [D] Working directory: D:\Games\ModOrganizer2 SkyrimSE
18:33:50 [D] Loaded settings:
18:33:50 [D]   archive_parsing_experimental=false
18:33:50 [D]   base_directory=D:/Games/ModOrganizer2 SkyrimSE/ModOrganizer2
18:33:50 [D]   colorSeparatorScrollbars=true
18:33:50 [D]   compact_downloads=false
18:33:50 [D]   containedColor=#400000ff
18:33:50 [D]   containsPluginColor=#400000ff
18:33:50 [D]   crash_dumps_max=5
18:33:50 [D]   crash_dumps_type=1
18:33:50 [D]   display_foreign=true
18:33:50 [D]   endorsement_integration=true
18:33:50 [D]   executable_blacklist=Chrome.exe;Firefox.exe;TSVNCache.exe;TGitCache.exe;Steam.exe;GameOverlayUI.exe;Discord.exe;GalaxyClient.exe;Spotify.exe
18:33:50 [D]   force_enable_core_files=true
18:33:50 [D]   hide_api_counter=false
18:33:50 [D]   hide_unchecked_plugins=false
18:33:50 [D]   language=en
18:33:50 [D]   load_mechanism=0
18:33:50 [D]   lock_gui=true
18:33:50 [D]   log_level=1
18:33:50 [D]   meta_downloads=false
18:33:50 [D]   offline_mode=false
18:33:50 [D]   overwritingArchiveFilesColor=#40ff00ff
18:33:50 [D]   overwritingLooseFilesColor=#40ff0000
18:33:50 [D]   overwrittenArchiveFilesColor=#4000ffff
18:33:50 [D]   overwrittenLooseFilesColor=#4000ff00
18:33:50 [D]   style=
18:33:50 [D]   use_prereleases=false
18:33:50 [D]   use_proxy=false
18:33:50 [D] initializing core
18:33:50 [D] usvfs log messages are written to C:/Users/jkleb/AppData/Local/ModOrganizer/SkyrimSE/logs/usvfs-2019-10-15_22-33-50.log
18:33:50 [D] Initializing VFS <mod_organizer_instance, 1, 1, C:/Users/jkleb/AppData/Local/ModOrganizer/SkyrimSE/crashDumps>
18:33:50 [D] Downloads after refresh: 0
18:33:50 [D] initialize plugins
18:33:50 [D] looking for plugins in D:\Games\ModOrganizer2 SkyrimSE\plugins
18:33:50 [D] loaded plugin "bsa_extractor.dll"
18:33:50 [D] loaded plugin "check_fnis.dll"
18:33:50 [D] loaded plugin "diagnose_basic.dll"
18:33:50 [D] loaded plugin "game_enderal.dll"
18:33:50 [D] loaded plugin "game_fallout3.dll"
18:33:50 [D] loaded plugin "game_fallout4.dll"
18:33:50 [D] loaded plugin "game_fallout4vr.dll"
18:33:50 [D] loaded plugin "game_falloutNV.dll"
18:33:50 [D] loaded plugin "game_morrowind.dll"
18:33:50 [D] loaded plugin "game_oblivion.dll"
18:33:50 [D] loaded plugin "game_skyrim.dll"
18:33:50 [D] loaded plugin "game_skyrimse.dll"
18:33:50 [D] loaded plugin "game_skyrimvr.dll"
18:33:50 [D] loaded plugin "game_ttw.dll"
18:33:50 [D] loaded plugin "inibakery.dll"
18:33:50 [D] loaded plugin "inieditor.dll"
18:33:50 [D] loaded plugin "installer_bain.dll"
18:33:50 [D] loaded plugin "installer_bundle.dll"
18:33:50 [D] loaded plugin "installer_fomod.dll"
18:33:50 [D] loaded plugin "installer_manual.dll"
18:33:50 [D] loaded plugin "installer_ncc.dll"
18:33:50 [D] loaded plugin "installer_quick.dll"
18:33:50 [D] loaded plugin "FNISPatches.py"
18:33:50 [D] loaded plugin "FNISTool.py"
18:33:50 [D] loaded plugin "pyCfg.py"
18:33:50 [D] loaded plugin "ScriptExtenderPluginChecker.py"
18:33:50 [D] loaded plugin "plugin_python.dll"
18:33:50 [D] loaded plugin "preview_base.dll"
18:33:50 [D] Downloads after refresh: 0
18:33:50 [D] no plugin splash
18:33:50 [D] managing game at D:\Games\Steam\steamapps\common\Skyrim Special Edition
18:33:50 [D] setting up configured executables
18:33:50 [D] configured profile: Default
18:33:50 [D] Checking Nexus API Key...
18:33:51 [D] initializing tutorials
18:33:51 [D] loaded language en
18:33:52 [D] Downloads after refresh: 0
18:33:52 [D] Downloads after refresh: 0
18:33:52 [D] D:\Games\ModOrganizer2 SkyrimSE\ModOrganizer2\profiles\Default\modlist.txt saved
18:33:52 [D] interface Ethernet seems to be up (address: fe80::705e:3724:c70c:1873%ethernet_32769)
18:33:52 [D] reading save games from E:/Users/jkleb/Documents/My Games/Skyrim Special Edition/Saves
18:33:52 [D] enable local saves: 0
18:33:52 [D] reading save games from E:/Users/jkleb/Documents/My Games/Skyrim Special Edition/Saves
18:33:52 [D] displaying main window
18:33:53 [D] login successful
18:34:55 [D] D:\Games\ModOrganizer2 SkyrimSE\ModOrganizer2\profiles\Default\plugins.txt saved
18:34:55 [D] D:\Games\ModOrganizer2 SkyrimSE\ModOrganizer2\profiles\Default\loadorder.txt saved
18:34:55 [D] D:\Games\ModOrganizer2 SkyrimSE\ModOrganizer2\profiles\Default\lockedorder.txt saved
18:34:58 [D] D:\Games\ModOrganizer2 SkyrimSE\ModOrganizer2\profiles\Default\initweaks.ini saved
18:34:58 [D] D:\Games\ModOrganizer2 SkyrimSE\ModOrganizer2\profiles\Default\lockedorder.txt saved
18:34:58 [D] Load Mechanism: Mod Organizer
18:34:58 [D] USVFS DLL Name: 
18:34:58 [D] enable local saves: 0
18:34:58 [D] Updating VFS mappings...
18:34:58 [D] D:\Games\ModOrganizer2 SkyrimSE\ModOrganizer2\profiles\Default\archives.txt saved
18:35:00 [D] VFS mappings updated <linked 212 dirs, 5 files>
18:35:00 [D] Spawning direct process < "D:/Games/Steam/steamapps/common/Skyrim Special Edition/skse64_loader.exe" , "" , "D:/Games/Steam/steamapps/common/Skyrim Special Edition" >
18:35:00 [D] Waiting for spawned process completion : skse64_loader.exe (7480)
18:35:00 [D] Waiting for usvfs process completion : SkyrimSE.exe (11136)
19:13:55 [D] Waiting for process completion successfull
19:13:57 [D] D:\Games\ModOrganizer2 SkyrimSE\ModOrganizer2\profiles\Default\lockedorder.txt saved
19:15:06 [D] save D:/Games/ModOrganizer2 SkyrimSE/ModOrganizer2/webcache/cookies.dat
19:15:06 [D] D:\Games\ModOrganizer2 SkyrimSE\ModOrganizer2\profiles\Default\initweaks.ini saved
19:15:06 [D] D:\Games\ModOrganizer2 SkyrimSE\ModOrganizer2\profiles\Default\lockedorder.txt saved
19:15:06 [D] Load Mechanism: Mod Organizer
19:15:06 [D] USVFS DLL Name: usvfs_x64.dll
19:15:07 [D] save D:/Games/ModOrganizer2 SkyrimSE/ModOrganizer2/webcache/nexus_cookies.dat

**Comment by LostDragonist** [comment]
We've seen problems with both Spybot and Avast in the past.  I don't know if this symptom matches but looking at disabling them would be the first thing to do.  I know you haven't had the problem in the past and you've probably used the programs for years but they constantly update and are a black box of doing weird stuff to your computer.

If you can get to the settings window of MO2, go to Workaround and click "reset window geometries".  Then restart MO.  That might also shake something loose.

**Comment by kimohoki** [comment]
Thanks LostDragonist. I reset window geometries and things are back to norman.
